javascript/builtins/Object.json

```
@@ -927,6 +927,58 @@
             }
           }
         },
+        "hasOwn": {
+          "__compat": {
+            "mdn_url": "https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/Object/hasOwn",
+            "spec_url": "https://tc39.es/proposal-accessible-object-hasownproperty/",
```

spec_url requires a fragment identifier: https://github.com/mdn/browser-compat-data/blob/main/schemas/compat-data-schema.md#the-__compat-object

```suggestion
            "spec_url": "https://tc39.es/proposal-accessible-object-hasownproperty/#sec-object.hasown",
```